service/src/test/java/org/databiosphere/workspacedataservice/controller/RecordControllerMockMvcTest.java

```diff
@@ -51,7 +53,7 @@ void createInstanceAndTryToCreateAgain() throws Exception {
 		UUID uuid = UUID.randomUUID();
 		mockMvc.perform(post("/{instanceId}/{version}/", uuid, versionId)).andExpect(status().isCreated());
 		mockMvc.perform(post("/{instanceId}/{version}/", uuid, versionId)).andExpect(status().isConflict()).andExpect(
-				result -> assertEquals("This schema already exists", result.getResponse().getErrorMessage()));
+				result -> assertEquals("This instance already exists", result.getResolvedException().getMessage()));
```

Comment:
use `instanceof` instead of looking at the message:
```suggestion
result -> assertTrue(result.getResolvedException() instanceof IllegalInstanceCreationException));
```